README.md

```diff
@@ -206,3 +207,9 @@ Please check out [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines on submitting
 For further discussion points and to connect with the team, join us on the Unity Multiplayer Networking Discord Server - Channel #dev-samples
 
 [![Discord](https://img.shields.io/discord/449263083769036810.svg?label=discord&logo=discord&color=informational)](https://discord.gg/FM8SE9E)
+
+## Feedback Form
```

Comment:
We could also add this to our readme.asset so that users would see it in the editor when they open the project

Reply:
> We could also add this to our readme.asset so that users would see it in the editor when they open the project

Agreed, we can. @SamuelBellomo feel free to decide if we include it in the .asset as well.

Reply:
pushed the change. It should look like this 
![Screen Shot 2022-10-03 at 11 56 29 AM](https://user-images.githubusercontent.com/71790295/193622995-bf29d519-f80e-4815-9a31-3986ac4feb74.png)